net-misc/seafile/seafile-7.0.4.ebuild

```diff
@@ -0,0 +1,43 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 )
```

Comment:
All good, but we are currently looking into dropping python2.7 support and I see upstream for this package has enabled python3 support. Could you look into updating to that?
https://github.com/haiwen/seafile/commit/a9252aca234bd44bd4ae579e0d045d150c6751a6

Reply:
Done.